CHANGELOG-4.0.md

```diff
@@ -6,14 +6,16 @@ in 4.0 versions.
 
 * 4.0.0 (xxxx-xx-xx)
 
- * Add `ruflin/Elastica` 3.x support.
+ * Add `ruflin/Elastica` 5.1 support.
```

Comment:
i would go with the composer name ruflin/elastica (lowercase) https://github.com/ruflin/Elastica/blob/07a770164b37716748aca1f19340cce48233229b/composer.json#L2